code/src/css/button.css

```diff
@@ -0,0 +1,17 @@
+button {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding: 10px;
+  border-radius: 40px;
+  background-color: lightgreen;
+  font-weight: 700;
+  font-family: 'Gochi Hand', cursive;
+  font-size: 24px;
+  color: purple;
+  border: 4px dotted purple;
+}
+
+button:hover {
+  cursor:url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg'  width='40' height='48' viewport='0 0 100 100' style='fill:black;font-size:24px;'><text y='50%'>🍹</text></svg>") 16 0,auto; /*!emojicursor.app*/
+}
+
```

Comment:
This.Is.Amazing! Love this 'little' detail! Will for sure take this with me to our next project!